apps/server/src/services/dictionaryService.ts

```diff
@@ -17,14 +17,14 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import { BadRequestError, ConflictError, NotFoundError } from 'common';
+import { Dictionary, DictionaryDocument, DictionaryDocumentSummary, Schema } from 'dictionary';
 import * as immer from 'immer';
+import { omit } from 'lodash';
 import logger from '../config/logger';
 import * as DictionaryRepo from '../db/dictionary';
 import { normalizeSchema, validate } from '../services/schemaService';
-import { Dictionary, Schema } from '../types/dictionaryTypes';
-import { BadRequestError, ConflictError, NotFoundError } from '../utils/errors';
-import { incrementMajor, incrementMinor, isGreater } from '../utils/version';
-import _ from 'lodash';
+import { VersionUtils } from 'dictionary';
```

Comment:
there is another import from `dictionary` in line 21, this could be consolidated there.
another one of those npm package imports I didn't know existed.
is this just a confusing "absolute" import syntax?